### PR TITLE
123 multiple tags no lastn limit

### PR DESCRIPTION
**Pull request by tima**

### CODE REVIEW

**Comment by jayallen** [comment]
Same request here.  I have to have more than that for the commit note

**Comment by tima** [comment]
This description was in Byrne's ticket. Relevant part:

> The database query has a limit applied to it, so the result set on has
> 5 elements in it. Period. However, the unique filter is causing one of
> those entries to be skipped. MT then reaches the end of the result set
> having on displayed 4 out of the 5 entries within it. Ideally, MT
> should iterate over entries until the limit is satisfied
> programmatically.
> 
> By deleting the limit argument, you force MT to do just this.

**Comment by jayallen** [comment]
Actually, that's a little too wordy :-).  I pulled up the ticket and found the perfect commit message:

> [#123 state:resolved] Fixed an issue where `lastn` and `limit` attributes of mt:entries produce the wrong result when multiple mt:entries tags are specified

That's all I was looking for at this point.  Just something to get this thing and all the rest of the checkins committed.
